backend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM amd64/python:3.9-slim
```

Comment:
Do we need the `amd64/` on this since we added `platform: linux/amd64` in the `docker-compose.yml`?

Reply:
good catch